src/kayenta/report/detail/reportExplanation.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+import { ICanaryState } from 'kayenta/reducers';
+import { ICanaryExecutionStatusResult } from 'kayenta/domain/ICanaryExecutionStatusResult';
+
+import './reportExplanation.less';
+
+interface IReportMetadata {
+  run: ICanaryExecutionStatusResult;
+}
+
+const getReason = (run: ICanaryExecutionStatusResult): string => {
+  if (run && run.result && run.result.judgeResult && run.result.judgeResult.score) {
+    return run.result.judgeResult.score.classificationReason;
+  }
+  return null;
```

Comment:
typescript has [some new(ish) fanciness](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) that can help _a lot_ with these deeply nested field lookups, similar to some constructs you may have seen in other languages, e.g.:

```suggestion
  return run?.result?.judgeResult?.score?.classificationReason ?? null;
```

Reply:
Thanks @erikmunson -- I like that better. Fixing.

Reply:
Chatted offline -- currently this is causing our pre-commit hook to fail. The guess is that prettier needs to be updated. I'm going to merge this down as-is for now and we can update this line once that's done.